os/using-environment-variables-in-systemd-units.md

```diff
@@ -0,0 +1,116 @@
+# Using Environment Variables In systemd Units
+
+## Environment directive
+
+Systemd has Environment directive which sets environment variables for executed processes. It takes a space-separated list of variable assignments. This option may be specified more than once in which case all listed variables will be set. If the same variable is set twice, the later setting will override the earlier setting. If the empty string is assigned to this option, the list of environment variables is reset, all prior assignments have no effect.
+
+### etcd2.service Unit Example
+
+Let's review etcd2.service [drop-in] unit example. [Cloud-config][cloud-config] compiles special `/run/systemd/system/etcd2.service.d/20-cloudinit.conf` [drop-in] unit file using options defined in "etcd2:" yaml section: 
```

Comment:
Connect this to the first paragraph by saying that etcd uses env vars for config and we provide them via a dropin file